python/dgl/nn/pytorch/conv/graphconv.py

```diff
@@ -25,6 +25,16 @@ class GraphConv(nn.Module):
     (i.e.,  :math:`c_{ij} = \sqrt{|\mathcal{N}(i)|}\sqrt{|\mathcal{N}(j)|}`),
     and :math:`\sigma` is an activation function.
 
+    If a weight tensor on each edge is provided, the weighted graph convolution is defined as:
+
+    .. math::
+      h_i^{(l+1)} = \sigma(b^{(l)} + \sum_{j\in\mathcal{N}(i)}\frac{e_{ji}}{c_{ij}}h_j^{(l)}W^{(l)})
```

Comment:
Is this formulation correct?  The definition of graph convolution is `D^{-1/2} A D^{-1/2}`, where `D[i, i] = sum(A[i, :])`, for weighted graphs.

I will extend the formulation above to bipartite graphs (blocks) with `D_{out}^{-1/2} A D_{in}^{-1/2}`, where `D_out[i, i] = sum(A[i, :])` and `D_in[j, j] = sum(A[:, j])`.

Reply:
The reasons that I implement it in this way are:

1. For scalar edge weight, the correct normalization asks for a reversed block to compute the `g.out_degrees` counterpart.
2. For multi-dimensional edge weight, there's no broadly accepted definition yet.

Therefore with the current form, users can insert an edge weight for simple edge weight computation (like masking edges), and pre-normalize the edge weight to customize the computation if necessary.